beacon-chain/p2p/discovery_test.go

```diff
@@ -80,13 +75,13 @@ func TestStartDiscV5_DiscoverAllPeers(t *testing.T) {
 	}
 
 	// Wait for the nodes to have their local routing tables to be populated with the other nodes
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(1 * time.Second)
```

Comment:
Can this 1 * time.Second be put into a const at the top of the file? Easier to find and change in the future if needed